static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -1,19 +1,21 @@
 define([
-    'common/utils/fastdom-promise',
     'qwery',
     'bean',
     'Promise',
+    'common/utils/fastdom-promise',
     'common/utils/config',
     'common/utils/mediator',
-    'lodash/functions/curry'
+    'common/modules/commercial/dfp/track-ad-load',
+    'lodash/functions/memoize'
 ], function (
-    fastdom,
     qwery,
     bean,
     Promise,
+    fastdom,
     config,
     mediator,
-    curry
+    trackAdLoaded,
+    memoize
 ) {
     // total_hours_spent_maintaining_this = 60
```

Comment:
done. although in all fairness to the spacefinder, the biggest problem lied in article-body-adverts 😜